odoo/addons/test_menus/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Test Menus',
+    'version': '1.0',
+    'category': 'Tools',
+    'description': """
+This module will test the main menus of Odoo.
+""",
+    'depends': ['lunch', ],
```

Comment:
are you sure you depend on lunch?

Reply:
No, it's only on dev purpose, that way, I have  a module to test :-)
I will remove that depends at the end.